libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+    project ("transportsecure sockets unit test")
```

Comment:
I see that many lines have been indented in the file. Can you please remove the indentation?

Reply:
the indent style is following the pattern from secure sockets
https://github.com/aws/amazon-freertos/blob/master/libraries/abstractions/secure_sockets/utest/CMakeLists.txt

Will change to follow CSDK as discussed

Reply:
fixed in 463e64114f55c99d1c198b80637c86c5ff726bca